src/midiEditorCommands.cpp

```diff
@@ -729,6 +729,23 @@ void postMidiSelectNotes(int command) {
 	outputMessage(s);
 }
 
+void postMidiSelectEvents(int command) {
+	HWND editor = MIDIEditor_GetActive();
+	MediaItem_Take* take = MIDIEditor_GetTake(editor);
+	int evtIndex=-1;
+	int count=0;
+	for(;;){
+		evtIndex = MIDI_EnumSelEvts(take, evtIndex);
+		if (evtIndex == -1)
+			break;
+		++count;
+	}
+	// fakeFocus = FOCUS_NOTE;
```

Comment:
I think this line needs some debate. Do we want to introduce a specific focus for events? I don't think so, as either ccs or notes are events.

I propose:

1. Is focus is set to CC or Note, leave it alone
2. If not, set it to note

Reply:
Your proposal sounds good to me.